gensim/models/keyedvectors.py

```diff
@@ -395,9 +395,8 @@ def add_one(self, key, vector):
         Warning: using this repeatedly is inefficient, requiring a full reallocation & copy,
         if this instance hasn't been preallocated to be ready fro such incremental additions.
 
-        returns: actual index used TODO: other param docs
+        returns: actual index used FIXME: other param docs
```

Comment:
I think it's better to make GitHub issues for TODOs and FIXMEs:

- Unless we're explicitly grepping for TODO/FIXME/whatever, this thing is essentially invisible to developers.
- This text will appear in our publicly readable docs. It's not a good look.

Reply:
In general yes. Explicitly grepping for FIXMEs will be a crucial task before 4.0.0 though – I suspect there will still be several.

(Not) appearing in public docs is a feature – we never want FIXMEs in any release code.